charts/service-deployment/README.md

```diff
@@ -24,6 +24,35 @@ _Note_: This should be a long running process - if you are looking for cron-base
 
 This chart won't solve for every possible option of deploying a service - it is meant to serve as an opinionated starting point to get something working decently well.  For more flexibility open a PR or fork this chart to suit your specific needs.
 
+### VPA RBAC Requirements
+
+If you enable VPA (`vpa.enabled: true`), you may need to ensure that the appropriate RBAC permissions are in place for the service account to manage VPA resources. This typically requires a ClusterRole with permissions to manage `verticalpodautoscalers` resources in the `autoscaling.k8s.io` API group, bound to your service account via a ClusterRoleBinding.
```

Comment:
Should we note that this should exist on the cluster prior to enabling. It can be deployed [through vertical-pod-autoscaler helm charts](https://github.com/snowplow-devops/helm-charts/blob/7a10b7b7ecc7c956dda3306155da3c23ce437e9e/charts/vertical-pod-autoscaler/templates/vpa-rbac.yaml#L47) or through [vertical-pod-autoscaler rbac](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler/deploy)